.gitignore

```diff
@@ -1,6 +1,7 @@
 /.clangd
 /compile_commands.json
 *.cmake
+!nanothreadConfig.cmake
```

Comment:
This file should never be committed to the .git repository, so including it here with a negation doesn't seem right to me.

Reply:
This one is hand-written: https://github.com/mitsuba-renderer/nanothread/blob/da0962ae32092b46207889a0cd82f54361c79011/nanothreadConfig.cmake

It's accessed directly at https://github.com/mitsuba-renderer/nanothread/pull/7/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR78